doc/manual/plugins_global/saveimage.rst

```diff
@@ -0,0 +1,14 @@
+.. _sec-plugins-global-saveimage:
+
+SaveImage (Save File)
+=====================
+
+This brings up :ref:`local-plugin-saveimage` local plugin from the global
```

Comment:
This documentation should be updated and also include the relevant contents from `plugins_local/saveimage.rst`.